### PR TITLE
BAH-1017 | Preview and onSave event error handling 

### PR DESCRIPTION
**Pull request by AlekhyaYalla**
https://bahmni.atlassian.net/browse/BAH-1017

### CODE REVIEW

**Comment by bsneha90** [comment]
Can you please raise the version to next patch?

File: src/helpers/formRenderer.js [hunk 1]
```diff
@@ -2,7 +2,8 @@ import { Container } from 'components/Container.jsx';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import ControlRecordTreeBuilder from 'src/helpers/ControlRecordTreeBuilder';
-import ScriptRunner from './scriptRunner';
+import ScriptRunner from './ScriptRunner';
```
Comment (bsneha90) [inline src/helpers/formRenderer.js:5]: Why have u chnaged this to captial S?? "ScriptRunner"?

File: test/helpers/formRenderer.spec.js [hunk 2]
```diff
@@ -4,7 +4,8 @@ import sinon from 'sinon';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { expect } from 'chai';
-import ScriptRunner from '../../src/helpers/scriptRunner';
+import ScriptRunner from '../../src/helpers/ScriptRunner';
```
Comment (bsneha90) [inline test/helpers/formRenderer.spec.js:7]: Why have u chnaged this to captial S?? "ScriptRunner"?

File: src/helpers/formRenderer.js (hunk 1 above)
Comment (AlekhyaYalla) [reply src/helpers/formRenderer.js:5]: Normally we changed, i can revert it back to small s.

File: test/helpers/formRenderer.spec.js (hunk 2 above)
Comment (AlekhyaYalla) [reply test/helpers/formRenderer.spec.js:7]: Normally we changed, i can revert it back to small s.

Comment (bsneha90) [reply test/helpers/formRenderer.spec.js]: Yes, it will break on other OS, as all OS is not case sensitive

Comment (AlekhyaYalla) [reply test/helpers/formRenderer.spec.js]: okay, added commit changing them

**Pull request merged**